### DISCUSSION TITLE
Issues mentioned are lost

### DISCUSSION THREAD

**Discussion by sanada08** [community/community#42285] [Projects and Issues]
### Select Topic Area

Question

### Body

We have almost 8 to 9 issues in track, suddenly all the issues are gone.

**Comment by queenofcorgis** [comment]
Hi @sanada08,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have removed the duplicate posts, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
